src/read/pe/mod.rs

```diff
@@ -16,6 +16,9 @@ pub use section::*;
 mod data_directory;
 pub use data_directory::*;
 
+mod delay_load_import;
+pub use delay_load_import::*;
```

Comment:
I'm inclined to keep it all in `import.rs` instead of creating a new module.

Reply:
sounds good, done!